drivers/bus/ancillary_bus.c

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Software based bus for Ancillary devices
+ *
+ * Copyright (c) 2019-2020 Intel Corporation
+ *
+ * Please see Documentation/driver-api/ancillary_bus.rst for
+ * more information
```

Comment:
more information. 

Sentence to end with full stop.

Reply:
changed